lib/alchemy-pg_search.rb

```diff
@@ -19,5 +15,13 @@ def self.is_searchable_essence?(essence_type)
     def self.searchable_essence_classes
       SEARCHABLE_ESSENCES.map { |k| "Alchemy::#{k.classify}".constantize }
     end
+    
+    def self.search(query, ability: nil)
+      Search.search query, ability: ability
```

Comment:
Ruby braucht zwar keine Klammern, aber es liest sich doch schöner:

```suggestion
      Search.search(query, ability: ability)
```